app/components/providers/copy_course_content_warning_component.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Providers
+  class CopyCourseContentWarningComponent < ApplicationComponent
+    include I18n
```

Comment:
Better to just call `I18n.t` instead of including all instance methods to this scope